source/table.cpp

```diff
@@ -41,7 +41,15 @@ spv_context spvContextCreate(spv_target_env env) {
   spvOperandTableGet(&operand_table, env);
   spvExtInstTableGet(&ext_inst_table, env);
 
-  return new spv_context_t{env, opcode_table, operand_table, ext_inst_table};
+  return new spv_context_t{
+      env, opcode_table, operand_table, ext_inst_table,
+      nullptr /* an empty MessageConsumer */
```

Comment:
Then again I see how later you can avoid constructing objects in the failure case.  So I can see the case on efficiency grounds....